packages/ethereum-contracts/scripts/deploy-framework.js

```diff
@@ -358,13 +365,14 @@ module.exports = eval(`(${S.toString()})({skipArgv: true})`)(async function (
     const deployCFAv1 = async () => {
         // @note Once we have the actual implementation for the hook contract,
         // we will need to deploy it and put it here instead of ZERO_ADDRESS
-        const hookContractAddress = ZERO_ADDRESS;
+        const hookContractAddress = cfaHookContract || ZERO_ADDRESS;
         console.log("CFA Hook Contract Address:", hookContractAddress);
 
         const agreement = await web3tx(
             ConstantFlowAgreementV1.new,
             "ConstantFlowAgreementV1.new"
-        )(superfluid.address, hookContractAddress);
+        // use 200k as default for CFA_HOOK_GAS_LIMIT
+        )(superfluid.address, hookContractAddress, ethers.utils.parseUnits("200000"));
```

Comment:
set it to 0s, it is unnecessary. Otherwise it confuses the codeChanged detector without special handling.